src/org/opensolaris/opengrok/configuration/messages/Messages.java

```diff
@@ -126,18 +133,33 @@ public static void main(String[] argv) {
             port = 2424;
         }
 
-        if (text == null) {
+        if ((type.compareTo("config") != 0) && (text == null)) {
             System.err.println("No text given");
             b_usage();
-            System.exit(3);
+            System.exit(1);
         }
 
         Message m = Message.createMessage(type);
 
         if (m == null) {
             System.err.println("Uknown message type " + type);
             b_usage();
-            System.exit(4);
+            System.exit(1);
+        }
+
+        if (type.compareTo("config") == 0) {
```

Comment:
moved the load to setTextFromFile() in Message at the cost of not checking the usage fully in Messages